src/webrequest.js

```diff
@@ -55,6 +55,10 @@ function onBeforeRequest(details){
     recordFrame(details.tabId, details.frameId, details.parentFrameId, details.url);
   }
 
+  if (type == "ping" && details.tabId < 0 ){
```

Comment:
> Are there ever ping requests that happen on visible tabs? Is there a reason to make this check?

Yes, `navigator.sendBeacon()` can be called anytime, not only after the `onunload` event. I've seen Bing, Google Search, Google Analytics, Youtube and Moatads sending pings on visible tabs.

In the current Chrome stable,  tabId for visible tabs are missing due to https://crbug.com/522124. But this is already fixed and will land on release soon.

> Also it might be having a comment which points to an explanation of what a ping request is, since this may be non obvious.

Sure, I will add the comments. Thanks for the feedback!